Cargo.toml

```diff
@@ -88,7 +88,7 @@ etcd-client = "0.12"
 fst = "0.4.7"
 futures = "0.3"
 futures-util = "0.3"
-greptime-proto = { git = "https://github.com/GreptimeTeam/greptime-proto.git", rev = "b1d403088f02136bcebde53d604f491c260ca8e2" }
+greptime-proto = { git = "https://github.com/QuenKar/greptime-proto.git", rev = "17cc60cf5664359bc8a81fa2ac6ee38c0c88d483" }
```

Comment:
Let's create a pull request in [greptime-proto](https://github.com/GreptimeTeam/greptime-proto/)

Reply:
Maybe there is no need to modify it, I have changed the proto version.